### ISSUE TITLE
Incomplete results for queries about user story history

### ISSUE THREAD

**Issue by rgson** [nephila/python-taiga#75]
When querying a user story's history, the returned results are incomplete for stories with a long history.

The reason is pagination and the cutoff appears to be 30 entries. The function `TaigaAPI.history.user_story.get` does not support the pagination API, making it impossible (as far as I know) to get the remaining records.

I've applied the following patch to my local installation to bypass the issue:
```diff
  class HistoryEntity(object):
      """
      HistoryEntity model
      """
      endpoint = 'history'
  
      def __init__(self, requester):
          self.requester = requester
  
      def get(self, resource_id):
          """
          Get a history element
  
          :param resource_id: ...
          """
          response = self.requester.get(
              '/{endpoint}/{entity}/{id}',
-             endpoint=self.endpoint, entity=self.entity, id=resource_id
+             endpoint=self.endpoint, entity=self.entity, id=resource_id,
+             paginate=False
          )
          return response.json()
```

I'll gladly submit a pull request for it, but it is a bit ad hoc and could potentially cause the same problems that led to the pagination API being implemented in the first place, should there be user stories with a ridiculous amount of history.

Let me know what you think.

**Comment by oderwat** [comment]
Just ran into the same problem! Please consider adding the PR #76. It is quite useless without!

**Comment by yakky** [comment]
Fixed in #76 it will soon be released
Thanks for reporting this
